src/AlphaCPU.cpp

```diff
@@ -1796,11 +1788,11 @@ int CAlphaCPU::virt2phys(u64 virt, u64 *phys, int flags, bool *asm_bit,
 
       // transfer execution to PALcode
       state.exc_addr = state.current_pc;
-      if (flags & VPTE) {
+      if constexpr (!!(flags & VPTE)) {
```

Comment:
I had to use bang-bang in a few places to keep stricter C++17 compiler happy. Generated code is identical.